arches_controlled_lists/src/arches_controlled_lists/components/tree/AddDeleteControls.vue

```diff
@@ -265,12 +279,49 @@ await fetchListsAndPopulateTree();
             </h3>
         </div>
         <div class="button-controls-container">
-            <Button
+            <SplitButton
                 class="list-button"
                 :label="$gettext('Add New List')"
                 :severity="shouldUseContrast() ? CONTRAST : PRIMARY"
+                :model="addNewListOptions"
+                :pt="{
+                    pcButton: {
+                        root: { style: { width: '100%', fontSize: 'inherit' } },
+                    },
+                }"
                 @click="createList"
             />
+            <Dialog
+                v-model:visible="importDialogVisible"
+                position="center"
+                :header="$gettext('Import Controlled Lists from SKOS File')"
+                :close-on-escape="true"
+                :modal="true"
+                :pt="{
+                    root: {
+                        style: {
+                            minWidth: '50rem',
+                            borderRadius: '0',
+                        },
+                    },
+                    header: {
+                        style: {
+                            background: 'var(--p-navigation-header-color)',
+                            color: 'var(--p-slate-50)',
+                            borderRadius: '0',
+                        },
+                    },
+                }"
+                :class="'import-dialog'"
+            >
+                <ImportList
+                    @imported="
+                        importDialogVisible = false;
+                        fetchListsAndPopulateTree();
```

Comment:
This could be a little expensive. Did you consider just pushing the new list(s) into the top of the tree a la #101? I guess this should be evaluated in tandem with the comment about whether the 201 Created response should return data.